transport-native-epoll/src/main/java/io/netty/channel/unix/DomainSocketChannel.java

```diff
@@ -30,4 +31,6 @@
 
     @Override
     DomainSocketChannelConfig config();
+
+    PeerCredentials peerCredentials() throws IOException;
```

Comment:
Just information:  Netty-all compiles and all tests run, this add to the interface does not effect anything in netty besides the EpollDomainSocketChannel and that class was updated.  I do understand it would affect anyone that has implemented the DomainSocketChannel interface directly, just providing as much information as possible so we can get this PR completed.